canvas_sdk/methods/accounts.py

```diff
@@ -19,8 +21,10 @@ def list_accounts(request_ctx, per_page=None, **request_kwargs):
         per_page = request_ctx.per_page
     path = '/v1/accounts'
     payload = {
-        'per_page' : per_page,
+        'per_page': per_page,
     }
+    if as_user_id:
+        payload['as_user_id'] = as_user_id
```

Comment:
why not add this to the payload above, it's ok to be there if it's none. that will keep the same pattern as all the other modules.